app/assets/stylesheets/page_layouts/feedback-form.css.scss

```diff
@@ -29,10 +29,18 @@
   padding: .5em .5em .5em .7em;
   min-height: 0; /* support: IE7 */
   font-size: 100%;
+  font-weight: $bold;
   background: white;
+  color: $base-link-color;
+
   &.ui-accordion-header-active {
     font-weight: bold;
-    border: 3px solid $brand-mint;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.